src/Ilios/CoreBundle/Form/Type/TermType.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Ilios\CoreBundle\Form\Type;
+
+use Ilios\CoreBundle\Form\DataTransformer\RemoveMarkupTransformer;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Class TermType
+ * @package Ilios\CoreBundle\Form\Type
+ */
+class TermType extends AbstractType
+{
+    /**
+     * @param FormBuilderInterface $builder
+     * @param array $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('title', null, ['required' => false, 'empty_data' => null])
+            ->add('vocabulary', 'tdn_single_related', [
+                'required' => false,
+                'entityName' => "IliosCoreBundle:Vocabulary"
+            ])
+            ->add('description', 'purified_textarea', ['required' => false])
```

Comment:
@saschaben thanks for the quick feedback. will play it safe then and disallow markup for the time being. we can always soften this rule up later.